sdk/communication/communication-network-traversal/CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-07-12)
```

Comment:
This can be left at unreleased. We can fill in this date when we know the release date for beta 2